packages/core/src/index.ts

```diff
@@ -52,6 +53,7 @@ import createStore from './store';
 import { EditableType, NativeFactory } from './types/editable';
 import { RootState } from './types/state';
 
+export { BackendFactory as DndBackend };
```

Comment:
I think we should get rid of this and import directly from the package where needed.

Reply:
problem is that you need to install `react-dnd-core` in your application just for that type. It is not exportet from `react-dnd`. I want to avoid make it more complicated for users.